build/build-mrt.yml

```diff
@@ -135,6 +135,56 @@ steps:
     platform: '$(buildPlatform)'
     configuration: '$(buildConfiguration)'
 
+- task: CmdLine@2
+  displayName: 'test MRT - UnitTest'
+  inputs:
+    script: |
```

Comment:
nitty nit: why the pipe + line break?

Reply:
I found it easier to read/see the actual command that gets executed, but I can make it all one line if you like.

Reply:
No, it's fine. It's not important of course. And there's no right or wrong here.